alf/algorithms/algorithm.py

```diff
@@ -1110,6 +1110,8 @@ def train_from_replay_buffer(self, update_global_counter=False):
           for every update in ``num_updates_per_train_iter``, the data will
           be shuffled and divided into
           ``buffer_size//(mini_batch_size * mini_batch_length)`` "mini-updates".
+          If ``mini_batch_length`` is None, then ``unroll_length`` will be used
```

Comment:
For off policy training, I think mini_batch_length needs to be explicitly set by the user. Using unroll_length is often a bad choice.

Reply:
> For off policy training, I think mini_batch_length needs to be explicitly set by the user. Using unroll_length is often a bad choice.

This is for replay_whole_buffer, and I think it's reasonable. I've disabled this for off-policy training